tests/hypothesis_strategies/acquisition.py

```diff
@@ -7,15 +7,23 @@
     ProbabilityOfImprovement,
     UpperConfidenceBound,
     qExpectedImprovement,
+    qLogExpectedImprovement,
+    qLogNoisyExpectedImprovement,
+    qNoisyExpectedImprovement,
     qProbabilityOfImprovement,
+    qSimpleRegret,
     qUpperConfidenceBound,
 )
 
 acquisition_functions = st.one_of(
     st.builds(ExpectedImprovement),
-    st.builds(ProbabilityOfImprovement),
     st.builds(qExpectedImprovement),
+    st.builds(qLogExpectedImprovement),
```

Comment:
Here now different story regarding order: shrinking. Can we somehow cleverly order them from `easy` to `complex`? If that's possible, would be good to have a short comment above that mentions it

Reply:
ok i will reorder these assuming the following complexity order
analytical -> q _> log -> noisy
but I dare you comment on alphabetical order then :D

Reply:
done please verify and resolve